query-planner-js/src/__tests__/buildPlan.test.ts

```diff
@@ -3810,7 +3810,7 @@ describe('Fed1 supergraph handling', () => {
 });
 
 describe('Named fragments preservation', () => {
-  it('works with nested fragments', () => {
+  it('works with nested fragments 1', () => {
```

Comment:
stray?
```suggestion
  it('works with nested fragments', () => {
```

Reply:
The reason I had this in here is that if you try to run a test by name, jest will actually run all tests for which that name is a subset. So if you run "test1", but a "test11" and "test12" exist, it will run all three. I was trying to debug this test, and got confused because another test would run immediately afterwards, and it took me a while to realize it was a separate execution.